src/models/pie.js

```diff
@@ -139,26 +162,26 @@ nv.models.pie = function() {
                     color: d3.select(this).style("fill")
                 });
             });
-            ae.on('mouseout', function(d, i) {
+            ae.on('mouseout', function (d, i) {
                 d3.select(this).classed('hover', false);
                 if (growOnHover) {
                     d3.select(this).select("path").transition()
                         .duration(50)
-                        .attr("d", arc);
+                        .attr("d", arcs[i]);
                 }
-                dispatch.elementMouseout({data: d.data, index: i});
+                dispatch.elementMouseout({ data: d.data, index: i });
             });
-            ae.on('mousemove', function(d, i) {
-                dispatch.elementMousemove({data: d.data, index: i});
+            ae.on('mousemove', function (d, i) {
+                dispatch.elementMousemove({ data: d.data, index: i });
```

Comment:
There are a ton of these spacing changes... they really distract from your actual code changes, would it be possible to submit the PR without all the spacing changes?